pkg/model/autoscaler_ticktock.go

```diff
@@ -96,6 +96,20 @@ func (asts *autoscalerTicktockStock) Add(entity simulator.Entity) error {
 		// do nothing
 	}
 
+	//calculate CPU utilization
+	countActiveReplicas := 0
+	cpuUtilizationValue := 0
+	for _, en := range asts.cluster.ActiveStock().EntitiesInStock() {
+		replica := (*en).(*replicaEntity)
+		cpuUtilizationValue += replica.currentCPUUtilization / replica.totalCPUCapacity
```

Comment:
Is `replica.currentCPUUtilization` is percentage [0-100] or a ratio [0-1]?
Is `replica.totalCPUCapacity` in millicores?

So what is the "type" of the resulting `cpuUtilizationValue`?  Why are we dividing by capacity?

Reply:
replica.totalCPUCapacity in millicores. replica.currentCPUUtilization is the number of millicores is occupied now. Probably I should rename replica.currentCPUUtilization to replica.occupiedCPUCapacity.

Reply:
cpuUtilizationValue is a ration [0-1]. I'll rename some variables and code'll be clearer.

Reply:
I changed code in this place. See e91267565b8ba92593c89fe969f1d65a35e99baf commit.